### ISSUE TITLE
BUG: Clicking on interface controls doesn't work in Ubuntu 23.04

### ISSUE THREAD

**Issue by willc0de4food** [beekeeper-studio/beekeeper-studio#1628]
**Describe the bug**
When attempting to click on interface controls (a tab for a table, to edit a value in the database, etc), the click goes "through" Beekeeper Studio, and registers in the app behind BS.

**To Reproduce**
1. Open Beekeeper Studio community edition in Ubuntu 23.04
2. Connect to a database, and open a couple tables
3. Click on a table to switch the one being currently viewed
5. Attempt to click on a field to edit a value
6. See error

**Expected behavior**
Clicking on an interface control should select that interface control.

**Screenshots / Gifs**
![beekeeper-error](https://github.com/beekeeper-studio/beekeeper-studio/assets/312095/278d29fe-4340-4bcb-b319-37349d8a76f6)

**Version Information (please complete the following information):**
 - OS: Ubuntu 23.04 guest running on VMWare Workstation 17 player on Windows host
 - App Version: 3.9.14 installed via Snap command line, and 3.9.17 installed via deb from GH releases
 - Database type and version: Postgres 9.6

**Additional context**
The app worked perfectly on my Ubuntu 22.10 vm. I upgraded the vm to 23.04 via the in-OS upgrade process, and am now experiencing these issues.

**Comment by not-night-but** [comment]
Hey! So when you first open the app you're able to interact with it? Like you can connect to a db and everything? Or are you using the keyboard for that? Are there any errors in the dev console?

**Comment by willc0de4food** [comment]
Hello! Yeah, so here's the weirdest part: I'm partially able to interact with the app. This issue doesn't occur on every tab for every table. I've also experienced the issue when attempting to grab the scrollbar to move horizontally across a table's fields. So it doesn't seem to be specific to any 1 action or any specific interface element. Funny enough, I'm having trouble getting to the console tab in the developer tools because the click is falling through to the window behind the app. Clicking anywhere within the inspector is actually falling through, and moving the BS app to a different part of the screen doesn't seem to help at all. All I can see at the moment is this: 
![image](https://github.com/beekeeper-studio/beekeeper-studio/assets/312095/0ba2e687-25ed-40af-a295-750aa3c2da51)

I'll update here if/when I can find out more.

**Comment by not-night-but** [comment]
Oh that is super weird. I do worry that this is not a beekeeper issue and instead something related to the virtual machine or the window manager. Like this could be some weird issue with the app somehow resizing, but the window manager not properly resizing behind the scenes. Idk these are just shots in the dark, I haven't seen anything like this and can't find anything related to it online so far. Will keep poking around.

If you can interact properly when it's a smaller window, are you able to get to the console ? With there being 7 errors and that many warnings, maybe there's something weird going on with the electron initialisation.

**Comment by willc0de4food** [comment]
Turns out the errors were related to queries I had ran, they weren't app errors. I've closed & re-opened a new window, and only see 1 warning popping up:
![image](https://github.com/beekeeper-studio/beekeeper-studio/assets/312095/1efdfd7e-cf84-48a3-a8b7-7b1e83dc51b9)

**Comment by not-night-but** [comment]
@willc0de4food hmmm okay. I'll see if I can reproduce this at some point and keep you posted. So sorry you're having so much trouble. In the meantime, is there no way that you can install the app on windows and expose the VM ports to the host machine? Or even run openssh on the VM?

**Comment by willc0de4food** [comment]
@not-night-but sounds like a plan. I do have it installed on my host machine, but my VM is VPN'd to access company resources which my host doesn't have access to. I could VPN on my host machine, but I enjoy the separation lol so for the time being, I'll just deal with using it in the smaller window 🙂

**Comment by not-night-but** [comment]
@willc0de4food Ah okay. I totally get that. So sorry again.

**Comment by willc0de4food** [comment]
@not-night-but no worries at all, I've tried a bunch of DB clients and think this one is the best! I appreciate the work put into it 🙂

**Comment by not-night-but** [comment]
@willc0de4food I'm so glad to hear that! Thanks for reporting this, and if you have any feedback, bugs or features, please feel free to give it.

**Comment by willc0de4food** [comment]
@not-night-but you are right that the issue isn't specific to BS but probably related to the window manager or something, I just noticed the same thing is happening in GitAhead
